### PR TITLE
Korean Update

### PR DESCRIPTION
**Pull request by teabarley**
Sorry for repeating pull request. I found out that my update was not saved to my fork so I completely deleted ko-kr.xaml and uploaded it again. looks like it works now.

### CODE REVIEW

**Comment by thesupersonic16** [comment]
This PR is currently missing one string, can you please add the `DiffUIMoved` string?

**Comment by teabarley** [comment]
Added!

File: HedgeModManager/Languages/ko-KR.xaml [hunk 1]
```diff
@@ -82,6 +84,7 @@
     <system:String x:Key="DiffUIModified">변경됨: {0}</system:String>
     <system:String x:Key="DiffUIRemoved" >제거됨: {0}</system:String>
     <system:String x:Key="DiffUIRenamed" >이름 변경됨: {0}</system:String>
+    <system:String x:Key="DiffUIMoved"   >이동됨: {0}</system:String>
```
Comment (thesupersonic16) [inline HedgeModManager/Languages/ko-KR.xaml]: There is a conflict here, can you confirm these changes are correct? Please see https://github.com/thesupersonic16/HedgeModManager/commit/b229b601080f4e0fb4c9a82fd6af44b6d4b36cd2 for more information.
```suggestion
    <system:String x:Key="DiffUIModified">변경됨</system:String>
    <system:String x:Key="DiffUIRemoved" >제거됨</system:String>
    <system:String x:Key="DiffUIRenamed" >이름 변경됨</system:String>
    <system:String x:Key="DiffUIMoved"   >이동됨</system:String>
```

**Review by thesupersonic16 — CHANGES_REQUESTED** [review]
There is a conflict that needs resolving, please see the attached comment.

**Comment by teabarley** [comment]
Thank you for correcting. It looks like my work was based on old branch, which uses {0}. I deleted it so it'll be fine now.

**Review by thesupersonic16 — APPROVED** [review]

**Pull request merged**